### ISSUE TITLE
missing opengraph/oembed attributes in docs pages?

### ISSUE THREAD

**Issue by jni** [napari/docs#469]
## 🐛 Bug

When I add a link to the API documentation in Discourse, the link preview gives me this warning:

https://napari.org/stable/api/napari.components.LayerList.html#napari.components.LayerList

<img width="673" alt="Screenshot of a framed webpage preview, showing the page title followed by the text: 'Sorry, we were unable to generate a preview for this web page, because the following oEmbed / OpenGraph tags could not be found: description, image'." src="https://github.com/user-attachments/assets/db3e4716-e1e8-40b3-9239-d3a645583fd4">

I actually find the Discourse link previews a little obnoxious 😂 but I prefer to disable them manually than to accidentally disable them by not using the correct standards for things. 😂 

I don't know what OpenGraph or oEmbed mean but if they are standard things expected of any webpage then we should figure out how to enable them on the API docs.

**Comment by melissawm** [comment]
This is a weird one... I thought this was automatically done. Maybe there's a sphinx/theme setting for it?

**Comment by jni** [comment]
Yeah. Note that I don't consider investigating this a high priority — just wanted to note it down in case it's trivial and/or someone has time to devote to it.

**Comment by psobolewskiPhD** [comment]
Close by
https://github.com/napari/docs/pull/667
